source/_integrations/heos.markdown

```diff
@@ -69,73 +67,69 @@ Use the sign-out service to sign the connected controller out of a HEOS account.
 
 You can play a HEOS favorite by number or name with the `media_player.play_media` service. Example service data payload:
 
-```json
-{
-  "entity_id": "media_player.office",
-  "media_content_type": "favorite",
-  "media_content_id": "1"
-}
+```yaml
+entity_id: media_player.office
+media_content_type: "favorite"
+media_content_id: "1"
 ```
 
-| Attribute              | Description
-| ---------------------- | ---------------------------------------------------------|
-| `entity_id`            | `entity_id` of the player
-| `media_content_type`   | Set to the value `favorite`
-| `media_content_id`     | The nubmer (i.e., `1`) or name (i.e., `Thumbprint Radio`) of the HEOS favorite
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | yes      | Can be a single player entity or a list of player entities. **Either `entity_id` or `area_id` is required.** |
+| `area_id`              | yes      | A single area or a list of areas with media players. **Either `entity_id` or `area_id` is required.** |
```

Comment:
We currently don't document `area_id` because it isn't used manually. Futhermore, it isn't "either", both or none can be provided.

Reply:
Okay, I probably wanted too much there. I have removed `area_id` and the wrong note.